packages/test-utils/package.json

```diff
@@ -41,7 +41,8 @@
     "whatwg-fetch": "^3.6.2"
   },
   "peerDependencies": {
-    "@testing-library/react": "^14.0.0"
+    "@testing-library/react": "^14.0.0",
+    "react": "18.2.0"
```

Comment:
Without React listed as a dependency here, yarn would prevent test-utils from resolving `react/jsx-runtime`, which would disable the new JSX transform.